.circleci/config.yml

```diff
@@ -70,6 +93,156 @@ jobs:
+    steps:
+      - checkout
+      - run:
+          name: 'Install Dockerize'
+          command: |
+            wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run:
+          name: 'Wait for Mongo'
+          command: |
+            dockerize -wait tcp://127.0.0.1:27017 -timeout 1m
+            sleep 10
+      - run:
+          name: 'Setup Mongo'
+          command: |
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox tap-tester.env
+            source tap-tester.env
+            wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add -
+            echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/5.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+            apt-get update
+            apt-get install -y mongodb-org-shell mongodb-mongosh mongodb-org
+            mongosh -u $TAP_MONGODB_USER -p $TAP_MONGODB_PASSWORD --authenticationDatabase admin --eval "rs.initiate({_id: \"rs0\", members: [{_id: 0, host: \"$TAP_MONGODB_HOST:$TAP_MONGODB_PORT\"}]})"
+      - run:
+          name: 'Setup virtual env'
+          command: |
+            pyenv local 3.5.6
+            python3 -mvenv /usr/local/share/virtualenvs/tap-mongodb
+            source /usr/local/share/virtualenvs/tap-mongodb/bin/activate
+            pip install -U 'pip<19.2' 'setuptools<51.0.0'
+            pip install .[dev]
+      - run:
+          name: 'pylint'
+          command: |
+            source /usr/local/share/virtualenvs/tap-mongodb/bin/activate
+            make test
+      - run:
+          name: "Unit Tests"
+          command: |
+            source /usr/local/share/virtualenvs/tap-mongodb/bin/activate
+            pip install pymongo==3.12.3
+            nosetests tests/unittests/
+      - run:
+          name: 'Integration Tests'
+          command: |
+            source tap-tester.env
+            mkdir /tmp/${CIRCLE_PROJECT_REPONAME}
+            export STITCH_CONFIG_DIR=/tmp/${CIRCLE_PROJECT_REPONAME}
+            source /usr/local/share/virtualenvs/tap-tester/bin/activate
+            pip install pymongo==3.12.3
+            selected_tests=$(circleci tests glob "tests/*{data,interrupt,open,restrict,view}*")
+            echo -e "Tests running in this batch:\n$selected_tests\n"
+            echo $selected_tests > ./tests-to-run
+            if [ -s ./tests-to-run ]; then
+              for test_file in $(cat ./tests-to-run)
+              do
+                run-test --tap=tap-mongodb $test_file
+              done
+            else
+              echo "ERROR - File not found or file is empty!" 1>&2
```

Comment:
For reference:

The `1` here is technically implied since `echo` writes to `&1` by default. So this could be `echo "hi" >&2`

Reply:
Removed.